x-pack/plugins/cloud_integrations/cloud_gain_sight/server/assets/gainsight_widget.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+/* @notice
+ * Portions of this code are licensed under the following license:
+ *
```

Comment:
As Jon mentioned https://github.com/elastic/kibana/pull/141132/files#r993811934. We need to add the license here so it's populated in the NOTICE file